File: QB_Challenges/PrepInsta/CPP/01_Getting_Started/05_GO_Two_Nums/GOOTN_Ternary.cpp
#include <iostream>
using namespace std;

void GOOTN(){
    int num1, num2;
    cout <<"Enter the first number: ";
    cin >>num1;
    cout <<"Enter the second number: ";
    cin >>num2;

    string result =
    (num1>num2) ?
    to_string(num1) + " is greater than " + to_string(num2) :
    to_string(num2) + " is greater than " + to_string(num1) ;

    cout << result;
}

int main(){
    GOOTN();
    return 0;
}

File: QB_Challenges/PrepInsta/JAVA/01_Getting_Started/05_GO_Two_Nums/GOOTN_Ternary.java
import java.util.Scanner;

public class GOOTN_Ternary {

    public static void GOOTN() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the first number: ");
        int num1 = sc.nextInt();
        System.out.print("Enter the second number: ");
        int num2 = sc.nextInt();
        sc.close();

        String result = (num1>num2) ?
        num1+" is greater than "+ num2 :
        num2+" is greater than "+ num1;

        System.out.println(result);

    
    }

    public static void main(String[] args) {
        GOOTN();
    }
    
}
